doc/man7/openssl-env.pod

```diff
@@ -0,0 +1,80 @@
+=pod
+
+=head1 NAME
+
+openssl-env - OpenSSL environment variables
+
+=head1 DESCRIPTION
+
+The OpenSSL libraries use environment variables to override the
+compiled-in default paths for various data.
+To avoid security risks, the environment is usually not consulted when
+the executable is set-user-ID or set-group-ID.
+
+=over 4
+
+=item B<CTLOG_FILE>
+
+Specifies the path to a certificate transparency log list.
+See L<CTLOG_STORE_new(3)>.
+
+=item B<OPENSSL>
+
+Specifies the path to the B<openssl> executable. Only used by
+the B<rehash> script.
+See L<openssl-rehash(1)/Script Configuration>.
+
+=item B<OPENSSL_CONF>
+
+Specfies the path to a configuration file.
+See L<openssl(1)> and L<config(5)>.
+
+=item B<OPENSSL_ENGINES>
+
+Specifies the directory from which dynamic engines are loaded.
+See L<openssl-engine(1)>.
+
+=item B<OPENSSL_MALLOC_FD>, B<OPENSSL_MALLOC_FAILURES>
+
+If built with debugging, this allows memory allocation to fail.
+See L<OPENSSSL_malloc(3)>.
+
+=item B<OPENSSL_MODULES>
+
+Specifies the directory from which cryptographic providers are loaded.
+See L<openssl-provider(1)>.
+
+=item B<OPENSSL_WIN32_UTF8>
+
+If set, then L<UI_OpenSSL(3)> returns UTF-8 encoded strings, rather than
+ones encoded in the current locale, and
+the L<openssl(1)> program also transcodes the command-line parameters
+from wide characters to UTF-8.
```

Comment:
1. This is only available on windows.
2. In both cases, it transcodes (is that a word?) from the current *code page* (there's no locale as we understand it on windows) to utf-8.

Reply:
fixup commit pushed. To transcode is to "convert (language or information) from one form of coded representation to another." :)  Think of it as bytestream as opposed to translating.

Reply:
You need to somehow change the sentence about the command line. "wide characters" is bollocks...

Reply:
really?  Looking at apps/lib/win32_init.c seems to be all about wide chars.

Reply:
`GetCommandLineW()` gets you the command line as wide characters, no matter the code page.  But the user doesn't know that we use `GetCommandLineW()`, nor do they need to know.  That's why this isn't just about wide characters.

Reply:
Okay, thanks for correcting my Win32 ignorance.  fixup commit pushed.